samples/tab-personal-quickstart/python/app.py

```diff
@@ -0,0 +1,84 @@
+from flask import Flask, send_from_directory
```

Comment:
Please update the copyright header on the missing files as shown below:
#Copyright (c) Microsoft Corporation. All rights reserved.
#Licensed under the MIT License.